src/main/scala/dotty/dokka/tasty/comments/wiki/Converter.scala

```diff
@@ -22,13 +22,13 @@ class Converter(val repr: Repr) {
   def convertBody(body: Body): dkkd.DocTag = {
     dkkd.P(
       collect {
-        body.blocks.foreach(emitBlock(_, isTopLevel = true))
+        body.blocks.foreach(emitBlock(_))
```

Comment:
100% nitpicking: could this be `.foreach(emitBlock)` ? :)

Reply:
Why yes, it could!